### PR TITLE
Not found tag  217

### PR DESCRIPTION
**Pull request by arinkulshi-skylight**
# Pull Request

## Description
https://github.com/CDCgov/IDWA/issues/217

- [x] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [x] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [x] I have updated the documentation, if applicable.
- [x] I have added or updated test cases to cover my changes, if applicable.
- [x] I have minimized the number of reviewers to include only those essential for the review.
- [x] I have notified teammates in the review thread to build awareness.

### CODE REVIEW

**Comment by arinkulshi-skylight** [comment]
I added some tests to this ticket to address some of the previous changes where I did not include tests. Let me know if you have any questions.

File: OCR/frontend/src/pages/AnnotateTemplate.tsx [hunk 1]
```diff
@@ -117,14 +127,6 @@ const AnnotateTemplate: React.FC = () => {
             </div>
             <span className="margin-left-1 text-normal">{item.name}</span>
           </div>
-          {!item.required && (
```
Comment (arinkulshi-skylight) [inline OCR/frontend/src/pages/AnnotateTemplate.tsx:120]: Here is main change for this ticket

File: OCR/frontend/src/components/FilePreview.test.tsx [hunk 2]
```diff
@@ -1,136 +0,0 @@
-import { fireEvent, render, waitFor } from '@testing-library/react'
```
Comment (zdeveloper) [inline OCR/frontend/src/components/FilePreview.test.tsx:1]: why was this removed? is it not used?

Comment (zdeveloper) [reply OCR/frontend/src/components/FilePreview.test.tsx:1]: oh I see, it just got moved

Comment (arinkulshi-skylight) [reply OCR/frontend/src/components/FilePreview.test.tsx:1]: It has been moved into the tests folder.

File: OCR/frontend/src/pages/AnnotateTemplate.tsx (hunk 1 above)
Comment (knguyenrise8) [reply OCR/frontend/src/pages/AnnotateTemplate.tsx:120]: ##gotcha

**Review by knguyenrise8 — APPROVED** [review]
